react/features/calendar-sync/middleware.js

```diff
@@ -79,6 +80,11 @@ StateListenerRegistry.register(
     /* listener */ (googleAPIState, { dispatch }) => {
         dispatch(setCalendarAPIState(googleAPIState));
     });
+StateListenerRegistry.register(
+    /* selector */ state => state['features/google-api'].profileEmail,
```

Comment:
It feels off to me that there is now duplicated state. I think you mentioned you weren't sure about the whole profileEmail thing. Why does the same state need to be in calendar and google api? Could the components needing the email look in multiple places for it instead, maybe through a selector/helper that knows where to look in redux?

Or maybe they're separate enough concerns to keep the two separate state entries?